.travis/php/install.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+
+set -ev
+
+composer install --no-interaction
+vendor/bin/codecept --version
```

Comment:
Why do we want to display the Codeception version here? Isnt this line useless?

Reply:
It was there, I'll change it.